db/migrate/20160307161402_add_scheduled_reports.rb

```diff
@@ -0,0 +1,13 @@
+class AddScheduledReports < ActiveRecord::Migration
+  def change
+    create_table :scheduled_reports do |t|
+      t.string :name, null: false
+      t.string :frequency, null: false
```

Comment:
yah your understanding is correct I guess I was imagining that for the cases you mentioned above you'd have preset values:

```
enum frequency: { daily: 0, weekly: , monthly: 2, every_other_week: 3, monday_wednesday_friday: 4 }
```

this isn't ideal, but I guess I am not sure how storing these values as a string is any better?